src/Compiler/Microsoft.AspNetCore.Razor.Language/test/RazorCodeDocumentTest.cs

```diff
@@ -8,40 +8,6 @@ namespace Microsoft.AspNetCore.Razor.Language;
 
 public class RazorCodeDocumentTest
 {
-    [Fact]
-    public void Ctor()
```

Comment:
These constructor tests are no longer needed. Everything uses `RazorCodeDocument.Create(...)` and the constructor is private.